<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>252</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>accuracy:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>252</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="neighbor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>k_neighbor:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conf_mat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Confusion_matrix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensitivity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>f1-score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>specificity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensitivity">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="specificity">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="f1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="roc_curve">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ROC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AUC:</string>
    </property>
   </widget>
   <widget class="QLabel" name="auc_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
